app/graph/types/annotation_type.rb

```diff
@@ -1,18 +1,16 @@
-AnnotationType = GraphqlCrudOperations.define_annotation_type('annotation', { content: 'str' }) do
-  field :project_media do
-    type ProjectMediaType
+class AnnotationType < AnnotationObject
+  field :annotation, String, null: true
 
-    resolve -> (annotation, _args, _ctx) { annotation.annotated_type == 'ProjectMedia' ? annotation.annotated : nil }
+  field :project_media, ProjectMediaType, null: true, resolve: ->(**_inputs) do
```

Comment:
TODO: Split this into more typical resolve function, since that's how we do other things.